SimonOkello/Questioner
# [ft #162951620] Modify UI

**What does this PR do?**
Adds admin user button to index.html file.

**Description of Task to be completed?**
Have the following achieved:
Make sure you UI flows by adding relevant links in the navigation bar

**How should this be manually tested?**
Clone this repo:
`https://github.com/SimonOkello/Questioner.git`
`cd Questioner`
`cd UI`
Open `index.html` in your favorite browser to see the changes
